cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -196,6 +198,18 @@ func validateAutoSpcMaxPool(spc *apis.StoragePoolClaim) error {
 	return nil
 }
 
+// validateBDCount validates the block device count in case of manual
+// provisioning
+func validateBDCount(spc *apis.StoragePoolClaim) error {
+	if isManualProvisioning(spc) {
+		minReqBDCount := blockdevice.DefaultDiskCount[spc.Spec.PoolSpec.PoolType]
```

Comment:
DefaultDiskCount should be used for min count, not for multiple
And, also, this need to be checked for pool on each node

Reply:
Removed